versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.5.0"
+  required_version = ">= 1.6.0"
```

Comment:
Looks like the terraform version got caught in the find/replace

Reply:
aah! thanks!